lib/components/molecules/Popover/Popover.story.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+// import { action } from '@storybook/addon-actions';
+import {
+  defaultVisiblePopover,
+  hiddenPopover,
+  ListPopover,
+  noCloseBtnPopover,
+} from './Popover.mock';
+
+// Import Styled Component to showcase variations
+import Popover, { PopoverVanilla } from '.';
+
+storiesOf('Molecules', module).addWithChapters('Popover', {
+  chapters: [
+    {
+      title: `Helloooooo`,
+      sections: [
+        {
+          sectionFn: () => (
+            <PopoverVanilla
+              {...hiddenPopover}
+              className={`hide-default-sample ${hiddenPopover.className}`}
+            />
+          ),
+          options: {
+            showSource: true,
+            allowSourceToggling: true,
+            showPropTables: true,
+            allowPropTablesToggling: true,
+          },
+        },
+      ],
+    },
+    {
+      title: 'Popover Variations',
+      sections: [
+        {
+          title: '[isVisible=true] variation',
```

Comment:
Title Should be contextual and description: Please name this as "By Default Open Variation"